Use of /bigobj compiler flag for imagebuf.cpp

Building on windows, I am seeing the following

```
C:\bld\openimageio_1520982480036\work\oiio-Release-1.8.8\src\libOpenImageIO\imagebuf.cpp : fatal error C1128: number of sections exceeded object file format limit: compile with /bigobj
```

The bigobj compilation flag is already used for [`/src/libOpenImageIO/CMakeLists.txt#L153`](https://github.com/OpenImageIO/oiio/blob/e6d722b4b93986ea3cd8d0ff13e82e7b76d0b089/src/libOpenImageIO/CMakeLists.txt#L153)

--- a/src/libOpenImageIO/CMakeLists.txt
+++ b/src/libOpenImageIO/CMakeLists.txt
@@ -151,6 +151,9 @@ endif ()
 
 if (MSVC)
     set_property (SOURCE imagebufalgo_pixelmath.cpp APPEND_STRING PROPERTY COMPILE_FLAGS " /bigobj ")
+    set_property (SOURCE imagebufalgo_addsub.cpp APPEND_STRING PROPERTY COMPILE_FLAGS " /bigobj ")
+    set_property (SOURCE imagebufalgo_muldiv.cpp APPEND_STRING PROPERTY COMPILE_FLAGS " /bigobj ")
+    set_property (SOURCE imagebuf.cpp APPEND_STRING PROPERTY COMPILE_FLAGS " /bigobj ")
 endif ()
 
 if (WIN32)

